page-creator/apps/creator-fe/src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
 <page-creator-dashboard>
   <page-creator-nav></page-creator-nav>
-  <main>cos bedzie tu</main>
+  <page-creator-w-wrapper></page-creator-w-wrapper>
 </page-creator-dashboard>
```

Comment:
co to znaczy 'w-wraper'

Reply:
w to prefix zeby oznaczyc ze to widget

Reply:
a skad ja mam wiedziec ze 'w' to widzet. PROSZE zmienic na widget-wraper

Reply:
:)